src/theme.ts

```diff
@@ -0,0 +1,51 @@
+import { Theme } from 'theme-ui'
+
+export const white = '#fff'
```

Comment:
I believe we can avoid exporting those colors, and add them into the theme with meaningful names